br.com.barros.DevWidgets.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "br.com.barros.DevWidgets",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "devwidgets",
+    "separate-locales": false,
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--share=network"
+    ],
+    "modules": [
+      {
+        "name": "devwidgets",
+        "buildsystem": "simple",
+        "only-arches": [
+          "x86_64"
+        ],
+        "build-commands": [
+          "mkdir -p DevWidgets",
+          "tar -xf DevWidgets-Linux-Portable.tar.gz -C DevWidgets",
```

Comment:
this should be made as source of type `archive` and let flatpak-builder extract it.